CONFIGURATION.md

```diff
@@ -1,29 +0,0 @@
-| rope.compress_objectdb           |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | bool     | False                                                                                   |
-| rope.automatic_soa               |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | bool     | True                                                                                    |
-| rope.soa_followed_calls          | The depth of calls to follow in static object analysis                                                                                                                                                                                                                                                                                                                                                                                                                                                                            | int      | 0                                                                                       |
-| rope.preform_doa                 | If `False` when running modules or unit tests 'dynamic object analysis' is turned off. This makes them much faster.                                                                                                                                                                                                                                                                                                                                                                                                               | bool     | True                                                                                    |
```

Comment:
typo: this should have been `perform_doa`

Reply:
fixed